README.md

```diff
@@ -12,14 +12,37 @@ curl -fsSL https://raw.githubusercontent.com/containers/ramalama/main/install.sh
 
 ## Usage
 
+### Listing Models
+
+You can `list` all models pulled into lcoal storage.
+
+```
+ramalama list
+```
 ### Pulling Models
 
-You can pull a model using the `pull` command. By default, it pulls from the ollama registry.
+You can `pull` a model using the `pull` command. By default, it pulls from the ollama registry.
 
 ```
 ramalama pull granite-code
 ```
 
+### Running Models
+
+You can `run` a chatbot on a model using the `run` command. By default, it pulls from the ollama registry.
+
+```
+ramalama run merlonite
```

Comment:
This doesn't exist for me, granite-code is an option

I did find a bug or something we have to enhance:

./ramalama pull instructlab/merlinite-7b-lab

this doesn't work, but it should, don't support non default namespaces like "instructlab"